openlibrary/templates/books/author-autocomplete.html

```diff
@@ -35,7 +35,7 @@
     <div class="input">
         <input name="$name_path--$i" class="author author-autocomplete" type="text" id="author-$i" value="$author.name"/>
         <input name="$dict_path--$i--author--key" type="hidden" id="author-$i-key" value="$author.key" />
-        <a href="javascript:;" class="remove red plain" title="$_('Remove this author')">[&times;]</a>&nbsp;
+        <a href="javascript:;" class="remove red plain" style="visibility: hidden;" title="$_('Remove this author')">[&times;]</a>&nbsp;
         <a href="javascript:;" class="add">$_("Add another author?")</a>
```

Comment:
This element needs to be hidden too. It depends on Javascript to work.

Reply:
@tfmorris understood. won't happen again